README.md

```diff
@@ -1,2 +1,43 @@
-# Math-Magicians
-"Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) created using React and allows users to make simple calculation and read a random math-related quote.
+# Math-Magician
+
+This is my Math-Magician project setup in the Microverse curriculum. "Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) created using React and allows users to make simple calculation and read a random math-related quote.
+
+## Built With
+
+- HTML
+- CSS
+- Javascript (Webpack)
+- React App
+- GitHub
+
+## Getting Started
+
+To get a local copy up and running:
+
+1. Clone this repository or download the Zip folder:
+
+**``git clone https://aburayhaan.github.io/Math-Magicians/``**
+
+1. Navigate to the location of the folder in your machine:
+
+**``you@your-Pc-name:~$ cd <folder>``**
+
```

Comment:
- [OPTIONAL] It will be nice to add explicit steps on how to clone and run your project locally, as this will be of help to non technical users and some of them are potential recruiters. Please kindly add explicit steps from cloning the app to seeing the results displayed on the browser.